config/default/kustomization.yaml

```diff
@@ -42,6 +42,11 @@ patchesStrategicMerge:
 # 'CERTMANAGER' needs to be enabled to use ca injection
 #- webhookcainjection_patch.yaml
 
+#  Uncomment the patches line if you enable Metrics and CertManager
```

Comment:
Bunch of deprecated fields like patchesStrategicMerge and vars are currently used by the kustomize and should be later updated.

Reply:
yeah, agree with you.